### PR TITLE
Remove the uitest module from the distro reference application

### PR DESCRIPTION
**Pull request by kdaud**
## What have done

1. I removed the ui-tests module since all the automated tests were migrated to [qaframework ](https://github.com/openmrs/openmrs-contrib-qaframework) module.

2. Also removed travis configurations since we migrated to using github actions.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ Vagrant.configure("2") do |config|
   # please see the online documentation at vagrantup.com.
 
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.box = "Ubuntu 12.04"
```
Comment (dkayiwa) [inline Vagrantfile:10]: How is this change related to what you are doing?

Comment (kdaud) [reply Vagrantfile:10]: Well, we  can have this change catered in a separate commit.

File: README.md [hunk 2]
```diff
@@ -1,55 +1,4 @@
 openmrs-distro-referenceapplication
 ===================================
 
-[master](https://github.com/openmrs/openmrs-distro-referenceapplication/) (platform-2.0.x): [![Build Status](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication/branches)
-
-[platform-2.1.x](https://github.com/openmrs/openmrs-distro-referenceapplication/tree/platform-2.1.x): [![Build Status](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication.svg?branch=platform-2.1.x)](https://travis-ci.org/openmrs/openmrs-distro-referenceapplication/branches)
-
-## Running with OpenMRS SDK
-
-Please follow the instructions at the [OpenMRS SDK Wiki page](https://wiki.openmrs.org/display/docs/OpenMRS+SDK). Set up a server with the Reference Application distribution.
```
Comment (sherrif10) [inline README.md:10]: Are we also getting away of using openMRS sdk. i think we still use this.

Comment (kdaud) [reply README.md:10]: The sub section of running with OpenMRS sdk was referring to running ui-test locally using a server set up via sdk. Now that these tests are no-longer in the module I think we can get rid of the section as well.

**Comment by kdaud** [comment]
Just a reminder to take another look at this PR after addressing the reviews. cc: @dkayiwa

**Comment by kdaud** [comment]
I think this commit is ripe for merging. cc: @dkayiwa

File: README.md [hunk 3]
```diff
@@ -1,55 +1,4 @@
...
-
-##  Running locally with Vagrant
-
-As of March 2017, the resources for this method are outdated/ no longer maintained.
-https://wiki.openmrs.org/x/CIC3Ag
-
-## Writing test cases 
-
-While writing the code for the test cases, please, follow the instructions in the [Code Style paragraph](https://wiki.openmrs.org/display/docs/Java+Conventions) and the [guidelines](https://wiki.openmrs.org/display/docs/Automated+Testing+Guidelines)
+This module contains configuration for the version of different modules used by RefApp 2.x, and 3.x distributions.
```
[the remark this answers is not in the document]
Comment (kdaud) [reply README.md]: Done!

**Comment by kdaud** [comment]
Hey @dkayiwa 

CI is not happy, there seems to be a job within Bamboo configured to checkout ui-test folder and now it's crying out for help.

**Comment by dkayiwa** [comment]
You can disable those tasks.
